tools/packager/Makefile

```diff
@@ -35,6 +38,16 @@ FPM_OPTS_RPM := -t rpm \
 	-d 'java >= 1:1.8.0' \
 	-d redhat-lsb-core
 
+
+EL6_PKG_PATH=target/marathon-$(PKG_VER)-$(PKG_REL).el6.noarch.rpm
+EL7_PKG_PATH=target/marathon-$(PKG_VER)-$(PKG_REL).el7.noarch.rpm
+UBUNTU_1404_PKG_PATH=target/marathon_$(PKG_VER)-$(PKG_REL).ubuntu1404_all.deb
```

Comment:
Didn't we drop Ubuntu 14.04 support?

Reply:
Yeah, we should just drop that one, it EOL'd in April